README.md

```diff
@@ -5,35 +5,12 @@ donations for the Common Voice project.
 ### Official Website
 [voice.mozilla.org](https://voice.mozilla.org)
 
-### Contribution
-[Non-code] Please help us add sentences to read. See [issue 341](https://github.com/mozilla/voice-web/issues/341) for details.
+### Contributing
+From adding sentences to read to enhancing our front end architecture, there are many ways to get involved with Common Voice. For more information, check out [CONTRIBUTING.md](https://github.com/mozilla/voice-web/blob/master/CONTRIBUTING.md).
```

Comment:
nitpick: there's a typo in here. should be something like:
```
From writing and reading sentences to enhancing our front-end architecture....
```

Reply:
No comment is too nitpicky! Thank you @mikehenrty